example/mapTiles.js

```diff
@@ -122,6 +125,17 @@ function initTiles() {
 
 	}
 
+	// listen to events to call render() on change
+	controls.addEventListener( 'change', render$ );
+	controls.addEventListener( 'end', render$ );
+	tiles.addEventListener( 'load-model', render$ );
+	tiles.addEventListener( 'force-rerender', render$ );
+
+	// workaround for initial loading
+	tiles.addEventListener( 'load-tile-set', render );
```

Comment:
Why do you feel this is a workaround? There's no reason to not debounce this, as well.